seeq/addons/plot_curve/backend/_equation.py

```diff
@@ -68,8 +68,9 @@ def fitted_function(self):
     @property
     @tracker(project=__name__)
     def seeq_formula(self):
-        splice = f"$splicedSignal = $signal.remove($signal.isNotBetween({str(min(self.x_data))}," \
-                 f"{str(max(self.x_data))})).convertUnits('{self.x_units}').setUnits('')"
+        conversion = f"$converted_signal = $signal.convertUnits('{self.x_units}').setUnits('')"
+        splice = f"$splicedSignal = $converted_signal.remove($converted_signal." \
+                 f"isNotBetween({str(min(self.x_data))}, {str(max(self.x_data))}))"
```

Comment:
pep8 prefers implied continuation inside parentheses over slash continuation: https://pep8.org/#:~:text=The%20preferred%20way%20of%20wrapping%20long%20lines%20is%20by%20using%20Python%E2%80%99s%20implied%20line%20continuation%20inside%20parentheses%2C%20brackets%20and%20braces.%20Long%20lines%20can%20be%20broken%20over%20multiple%20lines%20by%20wrapping%20expressions%20in%20parentheses.%20These%20should%20be%20used%20in%20preference%20to%20using%20a%20backslash%20for%20line%20continuation.

Reply:
Ah interesting... so in this case that would just be a ''' ''' instead of the \?